pkg/controllers/user/controllers.go

```diff
@@ -45,6 +47,7 @@ import (
 )
 
 func Register(ctx context.Context, cluster *config.UserContext, clusterRec *managementv3.Cluster, kubeConfigGetter common.KubeConfigGetter, clusterManager healthsyncer.ClusterControllerLifecycle) error {
+	setting.Register(ctx, cluster)
```

Comment:
do we really need to register setting controller in front of rbac controller?

Reply:
nope and updated.